pkg/destroy/openstack/openstack.go

```diff
@@ -15,7 +15,6 @@ import (
 	"github.com/gophercloud/gophercloud/v2/openstack/compute/v2/servergroups"
 	"github.com/gophercloud/gophercloud/v2/openstack/compute/v2/servers"
 	"github.com/gophercloud/gophercloud/v2/openstack/image/v2/images"
-	"github.com/gophercloud/gophercloud/v2/openstack/loadbalancer/v2/apiversions"
```

Comment:
@gryf The import change here requires a quick (see CI job [here](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_installer/9724/pull-ci-openshift-installer-main-verify-deps/1924435483429965824):

```
go mod tidy
go mod vendor
```